Doc/library/pyexpat.rst

```diff
@@ -867,6 +867,40 @@ The ``errors`` module has the following attributes:
 .. data:: XML_ERROR_SUSPEND_PE
 
 
+.. data:: XML_ERROR_RESERVED_PREFIX_XML
+
+   An attempt was made to
+   undeclare reserved namespace prefix ``xml``
+   or to bind it to another namespace URI.
+
+
+.. data:: XML_ERROR_RESERVED_PREFIX_XMLNS
+
+   An attempt was made to declare or undeclare reserved namespace prefix ``xmlns``.
+
+
+.. data:: XML_ERROR_RESERVED_NAMESPACE_URI
+
+   An attempt was made to bind the URI of one the reserved namespace
+   prefixes ``xml`` and ``xmlns`` to another namespace prefix.
+
+
+.. data:: XML_ERROR_INVALID_ARGUMENT
+
+   This should not be reported to Python applications.
+
+
+.. data:: XML_ERROR_NO_BUFFER
+
+   This should not be reported to Python applications.
```

Comment:
`XML_ERROR_NO_BUFFER` → [`MemoryError`](https://docs.python.org/3/library/exceptions.html#MemoryError)

Reply:
There seems to be a misunderstanding of `XML_ERROR_NO_BUFFER` here. it is not an out-of-memory situation but a situation where there is no buffer at all. And it won't be thrown in CPython because of how the current integration calls libexpat.